app/cronjob/trailingTrade/step/determine-action.js

```diff
@@ -280,6 +314,16 @@ const execute = async (logger, rawData) => {
   //  last buy price has a value
   //  and total balance is enough to sell
   if (canSell(data)) {
+    if (
+      _.isEmpty(await getGridTradeLastOrder(logger, symbol, 'sell')) === false
```

Comment:
In which situation this will happen?

Reply:
After the grid trade for selling order is placed, then the bot should start monitoring the order in the `ensure-grid-trade-order-executed` step. 

If the grid trade is found, then this `determine-action` step should not attempt to determine the action, because it can override the grid trade that must be monitored.,